server/channels/store/localcachelayer/channel_layer.go

```diff
@@ -408,6 +437,7 @@ func (s LocalCacheChannelStore) SaveMember(rctx request.CTX, member *model.Chann
 	if err != nil {
 		return nil, err
 	}
+	// For redis, directly increment member count.
```

Comment:
What does this comment (and the other ones like this) mean?

Reply:
It means that since Redis is a shared cache, now we can actually update the cache directly by incrementing the count, rather than invalidating it across all nodes in the cluster and let each node re-populate the new value.

Reply:
So is this a TODO? Because we're invalidating the cache right below the comment, right? Or am I missing something else?

Reply:
Yep, it's a TODO. Ticket already filed here: https://github.com/mattermost/mattermost/pull/27752#issuecomment-2257490365